tests/integration/pybaseball/test_statcast_batter.py

```diff
@@ -27,7 +27,7 @@ def test_statcast_batter() -> None:
     assert result is not None
     assert not result.empty
 
-    assert len(result.columns) == 89
+    assert len(result.columns) == 92
```

Comment:
should this be `CURRENT_SC_COLUMNS` instead of `92`? maybe `CURRENT_SC_COLUMNS` should be in `conftest.py` for access across all test files?